src/core/public/chrome/recently_accessed/recently_accessed_service.ts

```diff
@@ -81,10 +84,16 @@ export class RecentlyAccessedService {
       },
 
       /** Gets the current array of history items. */
-      get: () => history.get(),
+      get: () => history.get().filter((item) => !!item.workspaceId),
```

Comment:
will this impact workspace disabled case? I think we should check workspaceEnabled flag

Reply:
yes I forgot it here, thx for reminding!